lib/fortnox/api/repositories/order.rb

```diff
@@ -0,0 +1,33 @@
+require "fortnox/api/repositories/base"
+require "fortnox/api/models/order"
+
+module Fortnox
+  module API
+    module Repository
+      class Order < Fortnox::API::Repository::Base
+
+        CONFIGURATION = Fortnox::API::Repository::Base::Options.new(
```

Comment:
So this could be something like:

    Fortnox::API::Repository::Base::Options.new do
      uri '/orders/',
      json_collection_wrapper 'Orders',
      json_entity_wrapper 'Order',
      unique_id 'DocumentNumber',
      attribute_name_to_json_key_map { administration_fee_vat: 'AdministrationFeeVAT', edi_information: 'EDIInformation', freight_vat: 'FreightVAT', total_vat: 'TotalVAT', vat_included: 'VATIncluded' }
      keys_filtered_on_save [ :url ]
    end

Which would silence the warning about too many arguments. But would it be better or as good?